guide/C/ch_invest.xml

```diff
@@ -1403,143 +1540,466 @@ Income
   <sect1 id="invest-sell1">
     <title>Selling Shares</title>
 
-    <para>Entering an investment you are selling is done in the same way as
-    buying one (see <xref linkend="invest-buy-new2" />) except the total cost
-    of the transaction is entered in the
-    <emphasis>Sell</emphasis> column and the <emphasis>Shares</emphasis>
-    column is entered as a negative amount. The net proceeds from the sale
-    should be transferred from the stock account to your bank or brokerage
-    account.</para>
-
-    <para>The proper recording of the stock sale *must* be done using a split
-    transaction. In the split transaction, you must account for the profit (or
-    loss) as coming from an <emphasis>Income:Capital Gains</emphasis> account
-    (or <emphasis>Expenses:Capital Loss</emphasis>). To balance this income,
-    you will need to enter the stock asset
-    twice in the split. Once to record the actual sale (using the correct
-    number of shares and correct price per share) and once to balance the
-    income profit (setting the number of shares to 0 and price per share to
-    0).</para>
-
-    <note>
-      <para>In order to get <application>&app;</application> to commit this
-      zero-share, zero-price split to the transaction, you *must*
-      <keycap>Tab</keycap> out of the split. If you use the
-      <keycap function="enter">Enter</keycap> key, <application>&app;</application>
-      will convert the split into shares of the commodity.</para>
-    </note>
-
-    <para>In the split transaction scheme presented below, the symbol
-    NUM_SHARES is the number of shares you are selling, SELL_PRICE is the
-    price for which you sold the shares, GROSS_SALE is the total price for
-    which you sold shares, equal to NUM_SHARES*SELL_PRICE. PROFIT is the
-    amount of money you made on the sale. COMMISSIONS are the brokerage
-    commissions. NET_SALE is the net amount of money received from the sale,
-    equal to GROSS_SALE - COMMISSIONS.</para>
-
-    <table>
-      <title>Selling Stock using Split Transaction Scheme</title>
-
-      <tgroup cols="5">
-		<colspec colnum="1" colwidth="2*"/>
-		<colspec colnum="1" colwidth="2*"/>
-		<colspec colnum="1" colwidth="2*"/>
-		<colspec colnum="1" colwidth="2*"/>
-		<colspec colnum="1" colwidth="2*"/>
-        <tbody>
-          <row>
-            <entry><emphasis>Account</emphasis></entry>
+    <para>Entering an investment sale transaction is done in a similar way to
+      entering a buy transaction (see <xref linkend="invest-buy-new2" />) except
+      the proceeds of the transaction is entered in the <emphasis>Sell
+      </emphasis> column and the <emphasis>Shares</emphasis> column is entered
```

Comment:
"the Shares column is entered as a negative amount" is a bit awkward. How about "the amount entered in the Shares column is negative"?